### ISSUE TITLE
Move site publicness button to a checkbox?

### ISSUE THREAD

**Issue by patrickmj** [omeka/omeka-s#513]
With the new move of "Site Info" page covering the basics, it seems like a checkbox for "public" among the other settings is more intuitive than having the eye up at the top by View Delete Save

It's also currently a little weird that clicking on it toggles the public/not public styling, but that doesn't take effect until hitting save. The result is that it looks like it happens by ajax, but doesn't

**Comment by zerocrates** [comment]
As a counterpoint, maybe there's value in having the "publicness" interface be consistent across the places it appears?

**Comment by patrickmj** [comment]
Yeah, hadn't ignored that. It'll be weird either way. 

The second point about clicking it (which I added as an edit to the original issue, so we might have written past each other) might be a separate thing.

**Comment by zerocrates** [comment]
Yes, I hadn't seen the edit (those helpfully don't show up when you're on an issue page, though new comments do)

The "this doesn't immediately save" thing... yes that's probably a separate issue, as it goes to all the _other_ uses of this rather than this one for a site. I think to some degree it may be less important once we finally land the "warn you if you leave without saving" branch.

**Comment by zerocrates** [comment]
@patrickmj have you looked at this again (specifically the "it looks like AJAX" part) on a recent pull? The unsaved-changes stuff is in, so that should alleviate the problem.
